lib/Dialect/SPIRV/Serialization/Deserializer.cpp

```diff
@@ -368,6 +376,23 @@ LogicalResult Deserializer::processDecoration(ArrayRef<uint32_t> words) {
   return success();
 }
 
+LogicalResult Deserializer::processMemberDecoration(ArrayRef<uint32_t> words) {
+  // The binary layout of OpMemberDecorate is different comparing to OpDecorate
+  if (words.size() != 4) {
+    return emitError(unknownLoc, "OpMemberDecorate must have 4 operands");
+  }
+
+  switch (static_cast<spirv::Decoration>(words[2])) {
+  case spirv::Decoration::Offset:
+    memberDecorationMap[words[0]][words[1]] = words[3];
+    break;
+  default:
+    return emitError(unknownLoc, "unhandled OpMemberDecoration with op code: '")
```

Comment:
This is not opcode. What about "unhandled OpMemberDecoration case: "?

Reply:
Thanks, I'll fix it.